--- app/config/nginx/nginx.conf.tpl	2017-11-29 01:14:27.000000000 +0700
+++ new/app/config/nginx/nginx.conf.tpl	2018-03-30 02:05:47.000000000 +0700
@@ -43,6 +43,12 @@
     error_log      /dev/null;
   }

+  # Common bundles
+  location ~* ^/([a-zA-Z0-9]+)\-[0-9A-Za-z]+\.(js|js.map|css|css.map)$ {
+    add_header 'X-Content-Type-Options' 'nosniff' always;
+    alias {{VAR_DIR}}/$1.$2;
+  }
+
   location @app {
     access_log  {{LOG_DIR}}/nginx-access.log app buffer=32k;

